src/nvim/version.c

```diff
@@ -939,21 +939,11 @@ void list_version(void)
   version_msg(USR_VIMRC_FILE2);
   version_msg("\"\n");
 #endif  // ifdef USR_VIMRC_FILE2
-#ifdef USR_VIMRC_FILE3
-  version_msg(_(" 3rd user vimrc file: \""));
-  version_msg(USR_VIMRC_FILE3);
-  version_msg("\"\n");
-#endif  // ifdef USR_VIMRC_FILE3
```

Comment:
I don't think this should be removed, see the discussion in #2580.
That is, unless you have a different reason to remove it.